src/enchcracker/SimpleRandom.java

```diff
@@ -15,6 +15,16 @@ private int next() {
         return (int)(seed >>> 17);
     }
 
+    // always used
+    public int next8() {
+        return (int)((8 * (long)next()) >> 31);
```

Comment:
This can produce negative outputs. Correct and faster would be simply `next() >>> 28`